scripts/prepare-machine.ps1

```diff
@@ -63,6 +63,23 @@ if ($IsWindows) {
         # TODO - Support installing VS and necessary addins
         # TODO - Install CMake
         # TODO - Check for Windows preview
+        $NasmVersion = "2.15.05"
```

Comment:
You need to do this in more than just the Dev configuration. You need to do it in build too.